packages/lms/src/lib/styles/_toc.scss

```diff
@@ -2,7 +2,7 @@
   position: sticky;
   top: 1rem;
   overflow-y: auto;
-  height: calc(100vh - 10rem);
+  height: 80vh; /* Adjust to your preference */
```

Comment:
```suggestion
```
There shouldn't really be a height value at all here as far as I can see, but I don't know if this large gap is actually intentional for whatever reason.